### PR TITLE
Feature/appveyor

### PR DESCRIPTION
**Pull request by jenol**

### CODE REVIEW

**Comment by pH200** [comment]
Feel free to remove sample projects for now. I'll put it back later.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - ps: .\tools\publish.ps1
+
+artifacts:
+  - path: '**\*.nupkg'
+    name: NugetPackage
+  
+deploy:
+  provider: NuGet
+  api_key:
+    secure: g5JNOfRkIpNjLPxGk2yfmG+RLPRkhqT71N11Hfx9ieRb90bUnPl0grIJpmFvwz7k
```
Comment (pH200) [inline appveyor.yml:21]: Is this secret?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+before_build:
+  - nuget restore
```
Comment (pH200) [inline appveyor.yml]: dotnet restore

File: tools/publish.ps1 [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    nuget spec    
+    [xml]$nuspec = gc $nuspecPath -Raw
+    $meta = $nuspec.package.metadata
+    $meta.id = $name
+    $meta.title = $name
+    $meta.authors = "Agoda"
+    $meta.owners = "Agoda"
+    $meta.projectUrl = "https://github.com/agoda-com/net-loadbalancing"
+    $meta.version = $version
+    $meta.description = "$name $version"
```
Comment (pH200) [inline tools/publish.ps1]: Those meta can be set on csproj.

File: tools/publish.ps1 [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+    ls (Join-Path $projectPath "bin\Debug\netstandard2.0\") -Filter "$name.*" | % {
+        Append-FileNode $files $_ "lib\netstandard2.0"
+    }
+
+    [void]$nuspec.package.AppendChild($files)
+
+    $nuspec.Save($nuspecPath)
+
+    nuget pack $nuspecPath -Version $version
```
Comment (pH200) [inline tools/publish.ps1]: Let's use `dotnet pack -c Release -p:PackageVersion=$version`

File: appveyor.yml (hunk 1 above)
Comment (jenol) [reply appveyor.yml:21]: yes it the is encrypted value `Your NuGet API key should be encrypted using this tool: https://ci.appveyor.com/tools/encrypt.`

**Pull request merged**